Makefile

```diff
@@ -152,7 +152,7 @@ clean-e2e:
 		echo "config-backup.json not found" && sed -i'' -e 's|"DataSource": ".*"|"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s"|g' config/config.json
 
 emojis: ## Creates emoji JSON, JSX and Go files and extracts emoji images from the system font
-	SERVER_DIR=$(BUILD_SERVER_DIR) npm run make-emojis
+	npm run make-emojis $(ARGUMENTS)
```

Comment:
I have tried running this locally. Do you know how we can pass the arguments via make. Both of the below doesnt seem to work.
1. `make emojis --server-dir ../mattermost-server`
<img width="962" alt="image" src="https://user-images.githubusercontent.com/17708702/151826478-dc94e895-ffde-45db-83ee-b9fe6d201803.png">

2. `make emojis -- --server-dir ../mattermost-server`
<img width="962" alt="image" src="https://user-images.githubusercontent.com/17708702/151826633-ed4923e6-c263-4da7-9a8d-2d57bb271582.png">

Actually should this command be in the make file anyways @hmhealey ?

Reply:
make ARGUMENTS='--server-dir ../mattermost-server' emojis

Reply:
> Actually should this command be in the make file anyways @hmhealey ?

Yeah, let's keep it here for now. It might be nice to get rid of the Makefile one day to be more JS-friendly, but there's currently a lot of stuff here that doesn't fit in the package.json, so that'd require somewhere else to put that.